### PR TITLE
Parse where clause query param

### PR DESCRIPTION
**Pull request by umar006**
Separate query params that need to be included in WHERE clause SQL

### CODE REVIEW

**Comment by fathisiddiqi** [comment]
repositorynya dimana mar?

**Comment by fathisiddiqi** [comment]
klo clean itu biasanya

- handler
- service
- domain
- repository/model

File: internal/handlers/catHandlers.go [hunk 1]
```diff
@@ -78,10 +80,14 @@ func HandleAddNewCat(db *sql.DB) gin.HandlerFunc {
 			}
 		}
 
-		query := `INSERT INTO cats (id, created_at, name, race, sex, age_in_month, description, image_urls)
-		VALUES ($1, $2, $3, $4, $5, $6, $7, $8)
+		// TODO: delete after auth api finish
+		parsed, _ := uuid.Parse("e91ce26e-9a53-4c4f-b5b5-0cad1a61d82b")
```
Comment (fathisiddiqi) [inline internal/handlers/catHandlers.go:84]: ini kenapa data dummy?

Comment (umar006) [reply internal/handlers/catHandlers.go:84]: karena auth api-nya blm jdi

Comment (umar006) [reply internal/handlers/catHandlers.go:84]: krn api auth blm finish fat, jdinya kupke dummy dlu biar gk kena error foreign key

**Comment by umar006** [comment]
> klo clean itu biasanya
> 
> * handler
> * service
> * domain
> * repository/model

nah itu ak gk tau klo hrus kyak gitu strukturnya

File: internal/handlers/catHandlers.go (hunk 1 above)
Comment (fathisiddiqi) [reply internal/handlers/catHandlers.go:84]: oh ok2

**Pull request merged**